spec/lib/whitehall_importer/integrity_checker/body_text_check_spec.rb

```diff
@@ -15,25 +15,52 @@
       expect(integrity_check.sufficiently_similar?).to be true
     end
 
-    it "returns true even if there is a mismatch in inline atttachment URL filesize" do
+    it "returns true even if there is a mismatch in an inline attachment URL filesize - with an anchor tag" do
       proposed_body = %(
-        <p>
-          <span class="gem-c-attachment-link">
-            <a class="govuk-link" href="filename.pdf" target="_blank">Test File</a>
-            (<span class="gem-c-attachment-link__attribute"><abbr title="Portable Document Format" class="gem-c-attachment-link__abbr">PDF</abbr></span>,
-              <span class="gem-c-attachment-link__attribute">391 KB</span>, <span class="gem-c-attachment-link__attribute">9 pages</span>)
-          </span>
+        <span class="gem-c-attachment-link">
+          <a class="govuk-link" href="filename.pdf" target="_blank">Test File</a>
+          (
+            <span class="gem-c-attachment-link__attribute">
+              <abbr title="Portable Document Format" class="gem-c-attachment-link__abbr">PDF</abbr></span>,
+            <span class="gem-c-attachment-link__attribute">391 KB</span>,
+            <span class="gem-c-attachment-link__attribute">9 pages</span>
+          )
+        </span>
+      )
+
+      publishing_api_body = %(
+        <span class="attachment-inline">
+          <a href="/filename.pdf">Test File</a>
+          (
+            <span class="type">PDF</span>,
+            <span class="file-size">391KB</span>,
+            <span class="page-length">9 pages</span>
+          )
+        </span>
+      )
+
+      integrity_check = described_class.new(proposed_body, publishing_api_body)
+      expect(integrity_check.sufficiently_similar?).to be true
+    end
+
+    it "returns true even if there is a mismatch in an inline attachment URL filesize - without anchor tag" do
```

Comment:
Similarly I think we can call this just "attachment" and don't need to specify "without an anchor tag".  In reality, when this attachment is rendered there will be a link to the attachment.  It's just that in order to solve this issue we only need to focus on the metadata part of the component.  You can see the code for this here: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_attachment.html.erb#L35

Reply:
Thanks @emmabeynon - that helps.  Have updated both tests accordingly.